### PR TITLE
Improvement/navigation/cancellation

### PR DESCRIPTION
**Pull request by julia-andersson**
## [APP-XXX]

- New navigation for termination flow

Some things needs to be fixed:
- Process view 100% directly
- survey not opening url in browser (do this on separate improvement branch)

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/App/Sources/MainNavigationJourney.swift [hunk 1]
```diff
@@ -272,7 +273,35 @@ struct MainNavigationJourney: App {
...
+                    TerminationFlowNavigation(
+                        configs: [contractConfig],
+                        isFlowPresented: { terminationDismissType in
+                            contractsNavigationVm.terminationContract = nil
+                            switch terminationDismissType {
+                            case .none:
+                                break
+                            case .chat:
+                                contractsNavigationVm.isChatPresented = true
+                            case .openFeedback(let url):
```
Comment (julia-andersson) [inline Projects/App/Sources/MainNavigationJourney.swift:285]: @sladan-hedvig not really sure how to place this. Also not sure how to make it work for embedded browser. Does not work right now.

File: Projects/TerminateContracts/Sources/Navigation/TerminationFlowNavigationViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+                let preSelectedTerminationDate = store.state.terminationDateStep?.minDate.localDateToDate
+                return preSelectedTerminationDate ?? Date()
+            }
+        )
+        .navigationTitle(L10n.setTerminationDateText)
+        .embededInNavigation(options: [.navigationType(type: .large)])
+    }
+
+    private func openProgressScreen() -> some View {
+        ProcessingView<TerminationContractStore>(
```
Comment (julia-andersson) [inline Projects/TerminateContracts/Sources/Navigation/TerminationFlowNavigationViewModel.swift:231]: The progress bar is instantly set to 100%. Don't know why @sladan-hedvig

File: Projects/App/Sources/MainNavigationJourney.swift [hunk 3]
```diff
@@ -272,7 +273,35 @@ struct MainNavigationJourney: App {
...
                     let contractConfig: TerminationConfirmConfig = .init(contract: contract)
-                    TerminationViewJourney(configs: [contractConfig])
+                    TerminationFlowNavigation(
+                        configs: [contractConfig],
+                        isFlowPresented: { terminationDismissType in
+                            contractsNavigationVm.terminationContract = nil
+                            switch terminationDismissType {
+                            case .none:
+                                break
+                            case .chat:
```
Comment (julia-andersson) [inline Projects/App/Sources/MainNavigationJourney.swift:283]: this one is not working @sladan-hedvig

**Review by sladan-hedvig — APPROVED** [review]
I have some ideas, but thats more on global level. But will try it and present to you.

**Pull request merged**